packages/playground/website/src/components/site-manager-preview/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { JustViewport } from '../playground-viewport';
+import css from './style.module.css';
+
+export function SiteManagerPreview({
```

Comment:
I'm lost in all the SiteManager components. There is:

* Site manager editor
* Site manager preview
* Site manager sidebar
* Site manager

Could we co-locate all of that in a single directory, and perhaps merge `SiteManagerEditor`, `SiteManagerPreview`, and `SiteManager`? The sidebar seems substantial so let's keep it separate.

Reply:
I moved all site manager components under a single component. This matches what we did with GitHub components.

I removed the `SiteManagerPreview` component as it's not needed.

The `SiteManagerEditor` is currently a placeholder for the upcoming feature. I could remove it, but I don't see any harm in keeping it around.